pkg/controller/storagecluster/reconcile.go

```diff
@@ -1000,7 +1000,6 @@ func newStorageClassDeviceSets(sc *ocsv1.StorageCluster) []rook.StorageClassDevi
 				(&in).DeepCopyInto(&placement)
 
 				if len(topologyKeyValues) >= replica {
-					portable = true
```

Comment:
How about we also remove the variable on line 996 and then on line 1026 we use `ds.Portable` directly.

Reply:
I suppose we can, yeah. We can just introduce the variable again later if we have need.